src/config/babel.js

```diff
@@ -1,19 +1,29 @@
 import {promises as fsPromises} from 'fs';
 import {warn} from '@travi/cli-messages';
 
-export default async function ({projectRoot, preset, transpileLint}) {
+export default async function ({projectRoot, preset, transpileLint, tests}) {
   if (false === transpileLint) {
     warn('Not configuring transpilation');
 
     return {devDependencies: [], scripts: {}, vcsIgnore: {files: [], directories: []}};
   }
 
   if (preset) {
-    await fsPromises.writeFile(`${projectRoot}/.babelrc`, JSON.stringify({presets: [preset.name], ignore: ['./lib/']}));
+    await fsPromises.writeFile(
+      `${projectRoot}/.babelrc`,
+      JSON.stringify({
+        presets: [preset.name],
+        ignore: ['./lib/'],
+        ...tests.unit && {env: {test: {plugins: ['istanbul']}}}
+      })
+    );
   } else warn('Not configuring transpilation since no babel-config was provided');
 
   return {
-    devDependencies: ['@babel/register', ...preset ? [preset.packageName] : []],
+    devDependencies: [
+      '@babel/register',
```

Comment:
i know this was before your change, but it seems like `@babel/register` should consider the `preset` presence too. am i crazy thinking that?

Reply:
i could definitely see that making some sense. actually it looks like if we did go that route, that could be part of the top level early-out code so we could clean up the rest of it as well

Reply:
yeah, cool. some of these have gotten more complex over time, so some fresh eyes to clean up situations like that are always helpful.

totally cool with that cleanup being separate from this, but i'd like us not to lose track of it if possible